editor/translations/es.po

```diff
@@ -6018,7 +6022,7 @@ msgid "The path does not exist."
 msgstr "La ruta no existe."
 
 #: editor/project_manager.cpp
-msgid "Please choose a 'project.godot' file."
+msgid "Please choose a 'project.godot' or '.zip' file."
```

Comment:
`mgsid`s in translations should not be modified without fuzzying the string - as it is the `msgstr` no longer matches the English version, but translators won't see it as needing review.

But I'd suggest to remove the translation catalogs updates, as they are currently synced with the `3.0` code base on Weblate, so none of the `master` `TTR` changes have been propagated yet.

Reply:
Okay, so you would like to leave the message as it was originally and not include a reference to the fact that you can select a zip file as well now?

I also added the message "Invalid '.zip' project file, does not contain a 'project.godot' file" to all the translations as I felt it was a necessary message.  I added this by modifying all the files directly, is there another way I should do it so that translators are alerted to the new string?  Or, do you believe this change is unnecessary and the new message is not needed?

Reply:
No you can modify the string, but just not sync the change in `editor/translations/`.